tests/sympc/tensor/mpc_tensor_test.py

```diff
@@ -140,6 +140,25 @@ def test_ops_mpc_mpc(get_clients, nr_clients, op_str) -> None:
     assert np.allclose(result, expected_result, rtol=10e-4)
 
 
+@pytest.mark.parametrize("nr_clients", [2])
+@pytest.mark.parametrize("op_str", ["truediv"])
```

Comment:
NITPICK: Could you use ```div``` here?

Reply:
I guess we have only __truediv__  in python __div__ doesn't work